### PR TITLE
Win2019 Add windows 8.1 SDK

### PR DESCRIPTION
**Pull request by ghost**
# Description
Improvement - Add Windows 8.1 SDK to Win2019 image.

## Check list
- [ ] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Comment by maxim-lobanov** [comment]
Hello,
This won't work because Windows 8.1 SDK is not available in Visual Studio 2019 installer:

> The Windows 8.1 SDK is no longer available in the Visual Studio installer. Please upgrade your C++ projects to the latest Windows 10 SDK. If you have a hard dependency on 8.1, you can download it from the Windows SDK archive.

It can be installed separately but let's [discuss](https://github.com/actions/virtual-environments/issues/842#issuecomment-625062036) installation in issue before. We have some concerns about installation of legacy sdks since usually they are pretty huge

**Comment by maxim-lobanov** [comment]
Closing the PR since related issue was closed #842

**Comment by Edgarruiz8585** [comment]
> # Descripción
> Mejora: Añadir el SDK de Windows 8.1 a la imagen de Win2019.
> 
> #### Problema relacionado: #842
> ## Lista de verificación
> * [x] El problema/elemento de trabajo relacionado se adjunta.
> * [X] Las pruebas son escritas (si corresponde).
> * [X] La documentación se actualiza (si procede).
> * [X] Se prueban los cambios y se generan correctamente las imágenes de máquinas virtuales relacionadas.

 #  https://github.com/actions/runner-images/pull/845 @Edgarruuz8585
@misc{Edgar_Manuel_Ruiz_Arias_217596670

File: images/win/scripts/Installers/Windows2019/Install-VS2019.ps1
Comment (Edgarruiz8585) [inline images/win/scripts/Installers/Windows2019/Install-VS2019.ps1:1]: @misc{Edgar_Manuel_Ruiz_Arias_217596670

File: images/win/Windows2019-Readme.md
Comment (Edgarruiz8585) [inline images/win/Windows2019-Readme.md:1]: @misc{Edgar_Manuel_Ruiz_Arias_217596670

**Pull request closed without merging**